src/lamp_py/tableau/conversions/convert_gtfs_rt_vehicle_position.py

```diff
@@ -51,8 +51,58 @@ def lrtp(polars_df: pl.DataFrame) -> pl.DataFrame:
     """
     Function to apply final conversions to lamp data before outputting for tableau consumption
     """
+
+    def lrtp_restrict_vp_to_only_terminal_stop_ids(polars_df: pl.DataFrame) -> pl.DataFrame:
+        """
+        Function to apply lrtp filters conversions to lamp data before outputting for tableau consumption
+        """
+        stop_ids = list(map(str, [70110, 70162, 70236, 70274, 70502, 70510]))
+
+        #    pylint: disable=singleton-comparison
+        polars_df = polars_df.filter(~pl.col("vehicle.timestamp").is_null() & pl.col("vehicle.stop_id").is_in(stop_ids))
+        return polars_df
+
+    def temporary_lrtp_assign_new_trip_ids(polars_df: pl.DataFrame, threshold_sec: int = 60 * 15) -> pl.DataFrame:
+        """
+        Function to apply temporary trip ids to trips that have the same trip_id assigned for improbable stop durations at a single station
+
+        THIS IS NOT PERMANENT - THIS SHOULD BE REMOVED WHEN THE TRIP_IDS ARE PROPERLY POPULATED UPSTREAM
+
+        """
+        polars_df = (
+            polars_df.sort("feed_timestamp")
+            .with_columns(
+                pl.col("feed_timestamp").diff().alias("mdiff").over("vehicle.trip.trip_id", "vehicle.trip.start_date")
+            )
+            .with_columns(
+                pl.when(pl.col("mdiff") < threshold_sec)
+                .then(0)
+                .otherwise(1)
+                .fill_null(0)
+                .cum_sum()
+                .cast(pl.String)
+                .alias("new_id")
+                .over("vehicle.trip.trip_id", "vehicle.trip.start_date")
+            )
+            .with_columns(
+                pl.when(pl.col("new_id").ne("1"))
```

Comment:
so, you replace all trip IDs for a day except for one of them. why `.ne("1")` instead of `.ne("0")`? Wouldn't records with `new_id` = 0 be the first trip that day with that ID?